go/sql/types.go

```diff
@@ -44,6 +50,18 @@ func (this *Column) convertArg(arg interface{}) interface{} {
 		if encoding, ok := charsetEncodingMap[this.Charset]; ok {
 			arg, _ = encoding.NewDecoder().String(s)
 		}
+
+		if this.Type == BinaryColumnType {
+			size := len([]byte(arg.(string)))
```

Comment:
Minor nitpick: create a variable for `[]byte(arg.(string))` so that this doesn't need to evaluate twice.

Reply:
@cenkore could you please update code ^

Reply:
Sorry for the late reply，i'll update now.

Reply:
https://github.com/github/gh-ost/pull/915/commits/960d8012762f7140096f8727e0ac9e6e68f72636